### PR TITLE
Adjust name of `editor-searchable-dropdowns` and 3 other addons

### PR DESCRIPTION
**Pull request by DNin01**
### Changes

- **Changed "Grab single block with Ctrl key" to "Single block grabbing".** There are other addons such as "Muted project player mode" and "60FPS project player mode" with features that are activated with the keyboard without the names of those keys in their title, so this just makes these names more consistent. Everyone will see the description.

- **Changed "Duplicate script with Alt key" to "Quick script duplication".** Same reason.

- **Changed "Mouse position" to "Mouse position display"** to better describe what it does.

- **Changed "Search sprites by name" to "Sprite search"**
**and "Searchable dropdowns" to "Block dropdown search"** to make them sound more like feature names.

- **Changed "Search profiles" to "Search for users".** The first way sounded like you're searching for projects in profiles.

- **Updated the description of the "Block dropdown search" addon** to clarify that you can search for options in the dropdowns.

### Tests

Should work.

### CODE REVIEW

**Comment by WorldLanguages** [comment]
> Changed "Grab single block with Ctrl key" to "Single block grabbing"

Not convinced

> Changed "Search sprites by name" to "Sprite searching"

Good

**Comment by mxmou** [comment]
> * **Changed "Search sprites by name" to "Sprite searching"**
> * **and "Searchable dropdowns" to "Block dropdown searching".**

I think "Sprite search" and "Block dropdown search" would be better, for consistency with "Forum search". "Search profile" should probably be changed in a similar way.

File: addons/search-profile/addon.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Search profile",
+  "name": "Search for users",
```
Comment (Samq64) [inline addons/search-profile/addon.json:2]:
```suggestion
  "name": "User search",
```

Comment (DNin01) [reply addons/search-profile/addon.json:2]: Sounds more like you're searching projects a user made. That's why I changed it to "search _for_ users". Oh wait, that's more of a phrase than a noun, though. Hmm...

**Comment by WorldLanguages** [comment]
@DNin01 Could you elaborate the reasoning for each change in the first post of the PR?

**Comment by DNin01** [comment]
> Could you elaborate the reasoning for each change in the first post of the PR?

Description updated.

File: addons/editor-searchable-dropdowns/addon.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Searchable dropdowns",
-  "description": "Allows you to search block dropdowns.",
+  "name": "Block dropdown search",
+  "description": "Allows you to search for options in block dropdowns.",
```
Comment (RedGuy12) [inline addons/editor-searchable-dropdowns/addon.json:3]:
```suggestion
  "description": "Allows you to search the options in block dropdowns.",
```

File: addons/search-sprites/addon.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Search sprites by name",
+  "name": "Sprite search",
```
Comment (RedGuy12) [inline addons/search-sprites/addon.json:2]: This sounds more like a sharable sprite library. I prefer the old name over anything else I can think of

File: addons/search-profile/addon.json (hunk 1 above)
Comment (RedGuy12) [reply addons/search-profile/addon.json:2]: I like Search for users

**Review by RedGuy12 — CHANGES_REQUESTED** [review]

File: addons/search-sprites/addon.json (hunk 3 above)
Comment (DNin01) [reply addons/search-sprites/addon.json:2]: How about "search sprites in project"?

Comment (Samq64) [reply addons/search-sprites/addon.json:2]: `Project sprite search`?

File: addons/search-profile/addon.json (hunk 1 above)
Comment (DNin01) [reply addons/search-profile/addon.json:2]: Maybe "username search"?

File: addons/search-sprites/addon.json (hunk 3 above)
Comment (DNin01) [reply addons/search-sprites/addon.json:2]: I like that.
```suggestion
  "name": "Search box in sprite pane",
```

File: addons/search-sprites/addon.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Search sprites by name",
+  "name": "Search box in sprite pane",
```
Comment (Samq64) [inline addons/search-sprites/addon.json:2]:
```suggestion
  "name": "Sprite pane search",
```
The current name is fine too, I just like shorter ones.

**Review by Samq64 — APPROVED** [review]

File: addons/block-cherry-picking/addon.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Grab single block with Ctrl key",
-  "description": "Adds the ability to drag a single block out of the middle of a script (instead of the entire stack attached below it) while holding the Ctrl key.",
+  "name": "Single block grabbing",
+  "description": "Allows you to hold Ctrl to drag a single block out of the middle of a script (instead of the entire stack attached below it).",
```
Comment (WorldLanguages) [inline addons/block-cherry-picking/addon.json:3]: Does it make sense to change this one but not "Duplicate script with Alt key"?

File: addons/search-sprites/addon.json (hunk 4 above)
Comment (WorldLanguages) [reply addons/search-sprites/addon.json:2]: Personally I think "search box" is OK as it makes it obvious that this adds a new search bar.

File: addons/block-cherry-picking/addon.json (hunk 5 above)
Comment (DNin01) [reply addons/block-cherry-picking/addon.json:3]: I should probably change that one too.

**Comment by WorldLanguages** [comment]
I'm a bit worried about removing the keywords "Ctrl" and "Alt" from the addon names of block-cherry-picking and block-duplicate respectively. Could we open a different PR for those, and merge the other changes?

**Comment by DNin01** [comment]
@WorldLanguages Okay. I undid the name changes to both of the keyboard-activated addons.

**Review by WorldLanguages — APPROVED** [review]

**Pull request merged**